recipes/invoke/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set version = "0.12.2" %}
+
+package:
+  name: invoke
+  version: {{ version }}
+
+source:  
+  fn: invoke-{{ version }}.tar.gz
+  url: https://pypi.python.org/packages/source/i/invoke/invoke-{{ version }}.tar.gz
+  md5: f46192ea664325464a02f1c7f735e3cb
+
+build:
+  number: 0
+  script: python setup.py install
```

Comment:
It is unusual to use setuptools and not need to ```python setup.py install --single-version-externally-managed --record=record.txt```. Another option is to depend on pip and ```pip install --no-deps .```.